theano/gpuarray/tests/test_cgpukernelbase.py

```diff
@@ -0,0 +1,70 @@
+from __future__ import division, absolute_import, print_function
+import numpy
+from six.moves import xrange
+
+import theano
+from theano import tensor, config, Apply, Op
+from theano.gradient import grad_undefined
+
+from .config import mode_with_gpu, test_ctx_name
+
+from ..basic_ops import CGpuKernelBase
+from ..type import GpuArrayType, get_context
+
+
+from pygpu.gpuarray import dtype_to_typecode
+
+
```

Comment:
Can you add a comment (not in the docstring) explaining why this implementation is here?